Automattic/wp-calypso
# Subscribers Page: Create Empty List view

This PR creates the Empty List view for the Subscribers page.

Closes https://github.com/Automattic/wp-calypso/issues/77686

## Proposed Changes

* Create EmptyListView component
* Apply the created component to Subscribers page (client/my-sites/subscribers/main.tsx)
  * For now, it's always rendered at the first page
  * It will require a few changes to be displayed only for empty lists (when we have data on the page)

## Testing Instructions

* Apply this PR to your local env
* Run `yarn start`
* Go to http://calypso.localhost:3000/subscribers/<your-site>?flags=subscribers-page
* Verify if the Empty List view is rendered as expected (check on mobile screen sizes as well)

<img width="830" alt="image" src="https://github.com/Automattic/wp-calypso/assets/3113712/47d593e7-f3a9-4a76-86a8-643bd656802a">


## Pre-merge Checklist
